Assets/MixedRealityToolkit.Tests/PlayModeTests/SlateTests.cs

```diff
@@ -155,7 +181,7 @@ private IEnumerator RunGGVScrollTest(float expectedScroll)
             yield return h.Show(Vector3.zero);
             yield return h.Move(new Vector3(0.0f, -0.1f, 0f));
 
-            Assert.AreEqual(expectedScroll, totalPanDelta.y, 0.05, "pan delta is not correct");
+            Assert.AreEqual(expectedScroll, totalPanDelta.y, 0.1, "pan delta is not correct");
```

Comment:
Also out of curiousity, what was the delta with this change? i.e. because we changed the touch/scroll point mechanics I think that this tweak would be expected, I'm just curious how much of a delta it was compared to before.

Reply:
New Pan Delta is 0.2558216. On device this should remain the same, but on the editor, Hand Controller pointer (around Index Tip) is being used for initialization but Hand Palm position is being used on update, so there is an offset there. No Hand Palm on HL1.